### PR TITLE
setup basic structure, allow bert embedding

### PR DESCRIPTION
**Pull request by bdittes**
Adds basic dir structure for flask, package management, etc.

Implements BERT sequence to embedding inferrence.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "python.pythonPath": "venv/bin/python",
+    "python.linting.pycodestyleEnabled": true,
+    "python.linting.enabled": true,
+    "editor.tabSize": 4,
+    "editor.detectIndentation": false
+}
```
Comment (Tilana) [inline .vscode/settings.json]: This file stores visual studio code settings which should be unrelated to the topic classification repo. If so, let's add it to .gitignore

File: local.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import os
```
Comment (Tilana) [inline local.py]: from the name _local.py_ I don't really know what the file is doing. Maybe _main.py_ or _run.py_ is more informative?

File: .vscode/settings.json (hunk 1 above)
Comment (bdittes) [reply .vscode/settings.json]: You're right in principle, but I don't think there's anything wrong with keeping IDE setting in sync. I tried to keep them minimal (so they are only the vscode setting for making vscode work _with this repo_). There is not stricly a need to include autopep8 or ipython in the requirements.txt, except that they make development easier, so does this.

So I would prefer to keep them (uwazi also has .vscode/settings.json now :) )

File: local.py (hunk 2 above)
Comment (bdittes) [reply local.py]: In my understanding, this file is more like a test (the 'main' way to run this library is via server.py).

So I don't like main and run.py conflicts 'run'... but I'm open to any other name. I added a small doc to the top of the file.

File: .vscode/settings.json (hunk 1 above)
Comment (Tilana) [reply .vscode/settings.json]: okay, agreed.

File: local.py (hunk 2 above)
Comment (Tilana) [reply local.py]: The doc is helpful, thanks! Then let's keep the name.

**Pull request merged**